.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -16,14 +16,6 @@ body:
 
         > ℹ️ Please try to provide as much details as you can, the more details you can include the more likely it is for the issue to get quickly addressed.
 
-  - type: input
-    id: next-on-pages-version
-    attributes:
-      label: Cli version
-      description: Which version of `@cloudflare/next-on-pages` are you using?
-    validations:
-      required: true
-
```

Comment:
the next-on-pages version is included in the info below, so we shouldn't really ask this